screens/InventoryOrderFormScreen.js

```diff
@@ -338,13 +467,76 @@ class InventoryForm extends React.Component {
     constructor(props, context) {
         super(props, context)
         this.state = {
+            products: props?.products,
+            labels: props?.labels,
             openScanView: false,
+            openProductView: false,
             searchKeyword: props?.initialValues?.sku ?? '',
             searching: false,
             searchResults: [],
+            selectedSku: '',
+            selectedSkuLabel: '',
+            selectedSkuProduct: '',
             screenMode: 'SkuOnly',
-            inventoryId: null
+            inventoryId: null,
+            selectedToggleItems: new Map(),
+            collapsedId: '',
+            productId: null,
+            selectedProductName: '',
+            selectedProductInventory: null
         }
+
+        context.localize({
```

Comment:
It seems like you already defined these in global en/zh.js, you can remove them right?